src/algebra/homology/homological_complex.lean

```diff
@@ -176,6 +192,12 @@ end
   (f ≫ g).f i = f.f i ≫ g.f i :=
 rfl
 
+@[simp]
+lemma eq_to_hom_f {C₁ C₂ : homological_complex V c} (h : C₁ = C₂) (n : ι) :
+  homological_complex.hom.f (eq_to_hom h) n =
+  eq_to_hom (congr_fun (congr_arg homological_complex.X h) n) :=
+by { subst h, refl, }
```

Comment:
Again, what is this for? I'm generally pretty skeptical of developing too much theory around `eq_to_hom`!

Reply:
This one is used in order to get some equalities of functors, including this one :
`def map_alternating_face_map_complex {D : Type*} [category.{v} D] [preadditive D]
  (F : C ⥤ D) [F.additive] :
  alternating_face_map_complex C ⋙ (functor.map_homological_complex F _) =
  (simplicial_object.whiskering C D).obj F ⋙ alternating_face_map_complex D`

Reply:
Ok, I'm sold on this one.